test/studies/prk/stencil/PERFEXECOPTS

```diff
@@ -0,0 +1 @@
+--iterations=10 --order=1000
```

Comment:
Right that's what I meant, but I figured you might need to throw `--validate` in order to have something to search for. I could easily be wrong though